docs/orleans/grains/grain-lifecycle.md

```diff
@@ -53,12 +69,58 @@ public override void Participate(IGrainLifecycle lifecycle)
 
 In the above example, <xref:Orleans.Grain%601> overrides the <xref:Orleans.Grain.Participate%2A?displayProperty=nameWithType> method to tell the lifecycle to call its `OnSetupState` method during the <xref:Orleans.Runtime.GrainLifecycleStage.SetupState?displayProperty=nameWithType> stage of the lifecycle.
 
+<!-- markdownlint-disable MD044 -->
+:::zone target="docs" pivot="orleans-7-0"
+<!-- markdownlint-enable MD044 -->
+
+Components created during a grain's construction can take part in the lifecycle as well, without any special grain logic being added. Since the grain's context (<xref:Orleans.Runtime.IGrainContext>), including the grain's lifecycle (<xref:Orleans.Runtime.IGrainContext.ObservableLifecycle?displayProperty=nameWithType>), is created before the grain is created, any component injected into the grain by the container can participate in the grain's lifecycle.
```

Comment:
```suggestion
Components created during a grain's construction can take part in the lifecycle as well, without the addition of any special grain logic. Since the grain's context (<xref:Orleans.Runtime.IGrainContext>), including the grain's lifecycle (<xref:Orleans.Runtime.IGrainContext.ObservableLifecycle?displayProperty=nameWithType>), is created before the grain is created, any component injected into the grain by the container can participate in the grain's lifecycle.
```